modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/hero/index.css

```
@@ -14,6 +14,64 @@
 }
 
 .hero-container__image-hero {
-	border-radius: 0.75rem;
+	margin: 0px auto;
 	margin-top: 6rem;
+	width: 80%;
+}
+
+@media only screen and (max-width: 600px) {
+	picture {
+		display: flex;
+		justify-content: center;
+	}
+
+	.hero-container__image-hero {
+		margin-top: 6rem;
+		width: 80%;
+	}
+}
+
+@media only screen and (min-width: 600px) {
+	picture {
+		display: flex;
+		justify-content: center;
+	}
+	.hero-container__image-hero {
+		margin-top: 6rem;
+		width: 80%;
+	}
+}
+
+@media only screen and (min-width: 768px) {
+	picture {
+		display: flex;
+		justify-content: center;
+	}
+	.hero-container__image-hero {
+		margin-top: 6rem;
+		width: 80%;
+	}
+}
+
+@media only screen and (min-width: 992px) {
+	picture {
+		display: flex;
+		justify-content: center;
+	}
+	.hero-container__image-hero {
+		margin-top: 6rem;
+		width: 80%;
+	}
+}
+
+@media only screen and (min-width: 1200px) {
+	picture {
+		display: flex;
+		justify-content: center;
+	}
+
+	.hero-container__image-hero {
+		margin-top: 6rem;
+		width: 80%;
+	}
```

Shouldn't we try to make it more generic? We're using the same style for all screen sizes. 

Try to use something like 
```suggestion
  @media only screen and (min-width: 600px) and (max-width: ?px)  {
    //Put your CSS here//
   
    }

```